pkg/util/podgroup.go

```diff
@@ -27,7 +27,7 @@ import (
 	"sigs.k8s.io/scheduler-plugins/pkg/apis/podgroup/v1alpha1"
 )
 
-// DefaultWaitTime is 60s if MaxScheduleTime is not specified.
+// DefaultWaitTime is 60s if ScheduleTimeoutSeconds is not specified.
```

Comment:
Forgot to mention that a good practice is to use `Defaults_XYZ` logic to define the expected defaulting behavior, and register the the function to `RegisterDefaults()` (in pkg/apis/podgroup/v1alpha1/zz_generated.defaults.go).

Feel free to do it in this PR, or add a TODO here and accomplish in a follow-up.

Reply:
I would like  add a TODO here.

Reply:
PR updated